xml/security_ldap_replication.xml

```diff
@@ -0,0 +1,645 @@
+</screen>
+  <para>
+   Configure RW2:
+  </para>
+<screen>&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication create-manager</command>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> replication enable</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \ 
+<command>--role supplier --replica-id 2 --bind-dn "cn=replication manager,cn=config"</command>
+</screen>
+  <para>
+   This will create the replication metadata required on RW1 and RW2. Note the
+   difference in the <option>replica-id</option> between the two servers. This
+   also creates the replication manager account, which is an account with
+   replication rights for authenticating between the two nodes.
+  </para>
+  <para>
+   RW1 and RW2 are now both configured to have replication metadata. The next
+   step is to create the first agreement for outbound data from RW1 to RW2.
+  </para>
+<screen>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> repl-agmt create</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \
+<command>--host &exampleserverip; --port 3389 --conn-protocol LDAP --bind-dn "cn=replication manager,cn=config"</command> \
+<command>--bind-passwd <replaceable>PASSWORD</replaceable> --bind-method SIMPLE <replaceable>RW1_to_RW2</replaceable></command>
+</screen>
+  <para>
+   Data will not flow from RW1 to RW2 until after a full synchronization
+   of the database, which is called an initialization or reinit. This will
+   reset all database content on RW2 to match the content of RW1. Run the
+   following command to trigger a reinit of the data:
+  </para>
+<screen>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> repl-agmt init</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable> <replaceable>RW1_to_RW2</replaceable></command>
+</screen>
+  <para>
+   Check the status by running this command on RW1:
+  </para>
+<screen>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> repl-agmt init-status</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable> <replaceable>RW1_to_RW2</replaceable></command>
+</screen>
+  <para>
+   When it is finished, you should see a "Agreement successfully initialized"
+   message. If you get an error message, check the errors log. Otherwise, you
+   should see the identical content from RW1 on RW2.
+  </para>
+  <para>
+   Finally, to make this bi-directional, configure a replication agreement 
+   from RW2 outbound to RW1:
+  </para>
+<screen>
+&prompt.sudo;<command>dsconf <replaceable>INSTANCE-NAME</replaceable> repl-agmt create</command> \
+<command>--suffix dc=<replaceable>example</replaceable>,dc=<replaceable>com</replaceable></command> \
```

Comment:
Isn't it the whole suffix string that's a replaceable, not just the components? IE should be `--suffix <replaceable>dc=example,dc=com</replaceable>`

Reply:
it is!